test/plenario2/core/virtual_point_field_actions_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule VirtualPointFieldActionsTests do
+  use ExUnit.Case, async: true
+  alias Plenario2.Core.Actions.{VirtualPointFieldActions, MetaActions, UserActions}
+  alias Plenario2.Repo
+
+  setup do
+    :ok = Ecto.Adapters.SQL.Sandbox.checkout(Repo)
+  end
+
+  test "create virtual point field from long/lat" do
+    {:ok, user} = UserActions.create_user("Test User", "password", "test@example.com")
```

Comment:
I think you should use the `setup` callback here and pass the `user` and `meta` into the test's context. It'll save ya lots of lines. https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html